.gitmodules

```diff
@@ -1,12 +1,12 @@
 [submodule "src/sentry-dotnet"]
 	path = src/sentry-dotnet
 	url = https://github.com/getsentry/sentry-dotnet.git
-[submodule "src/sentry-cocoa"]
+[submodule "modules/sentry-cocoa"]
```

Comment:
odd, `main` branch is already on modules:

https://github.com/getsentry/sentry-unity/blob/9b0b81ae436d99968aa718a1c5d47e1898a009ea/.gitmodules#L4-L12

Reply:
nope, it's the "name" not path, see line *4*